src/OrchardCore.Modules/OrchardCore.Forms/Models/InputPart.cs

```diff
@@ -1,11 +1,20 @@
+using System.ComponentModel;
 using OrchardCore.ContentManagement;
 
 namespace OrchardCore.Forms.Models
 {
     public class InputPart : ContentPart
     {
         public string Type { get; set; }
+
         public string DefaultValue { get; set; }
+
         public string Placeholder { get; set; }
+
+        public LabelOptions LabelOption { get; set; }
```

Comment:
Similar to `FormElementPart` and `FormInputElementPart` let's have this moved new part `FormElementLabelPart` and provide views for it.

Let's keep `InputPart` and `SelectPart` unchanged and  for label add `FormElementLabelPart` to content type Input and Select using migration